ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/user_management_service.proto

```diff
@@ -98,6 +109,15 @@ message User {
     // Users for participant administrators MAY have an associated primary party.
     // Optional
```

Comment:
```suggestion
    // Optional, Read-Write
```

Reply:
@pbatko-da : please double-check that the 'Read-Write' annotations work; and explain them on the `UpdateUser` rpc.

Reply:
Yeah, I mean add an explanation of these at the top of the file, and then review whether the comments match the definition.

Reply:
Ok, done (but for now placed the explanation at the service level)